### PR TITLE
Update CONTRIBUTING.rst information wrt Slack channels

### PR DESCRIPTION
**Pull request by pankajkoti**
Update the renamed first PR support channel and add caution 
tip suggestubg to avoid cross-posting same question across 
multiple channels.

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -1466,6 +1466,9 @@ We don't create new issues on JIRA anymore. The reason we still look at JIRA iss
    * newbie questions (#newbie-questions channel)
```
Comment (eladkal) [inline CONTRIBUTING.rst]: newbie-questions channel does no longer exist

Comment (pankajkoti) [inline CONTRIBUTING.rst]:
```suggestion
```

Comment (pankajkoti) [reply CONTRIBUTING.rst]: may I know please why we archived that channel? :)

Comment (eladkal) [reply CONTRIBUTING.rst]: low traffic and most of times cross-posting in random or troubleshooting

**Review by eladkal — APPROVED** [review]

**Review by potiuk — APPROVED** [review]

**Pull request merged**